ci/tests/e2e-helm.sh

```diff
@@ -30,18 +30,42 @@ make install
 
 deploy_operator() {
     kubectl create -f "$OPERATORDIR/deploy/service_account.yaml"
+    if oc api-versions | grep openshift; then
+        oc adm policy add-cluster-role-to-user cluster-admin -z nginx-operator || :
```

Comment:
we shouldn't need to make the operator cluster-admin, is there something wrong with the permissions in role.yaml?

Reply:
This is the aws test which is executed https://github.com/openshift/release/pull/7420/files#diff-0fe368282e6b9e295b3096e55726040eR110

Note that it exactly equals the test performed for ansible which is adding this permission. Then, it has been facing issues in the CI of openshift/release/ and I believe that it will solve it.

Reply:
oh jeez you're right, well I don't like that but seems like it's a separate issue :+1: